_posts/2023-02-24-community-survey

```diff
@@ -0,0 +1,23 @@
+---
+layout: blog
+title: 2023 Community Survey
+author: nick
+description: It is time to re-launch our Node-RED community survey
+---
+
+Back in 2019 we published the results of the 2019 Node-RED community survey. The results 
+painted an interesting picture of the Node-RED community. We think it is time to refresh
+these results so we are running the survey again.
+
+The 2023 survey is similar to the 2019, we wanted to get a better picture of who our uses are, what they do with Node-RED
+and to make sure we keep the project pointed in the right direction.
+
+The survey will be open until March 31. Of course we would like to get as many respondents
+as possible. In 2019, we have over 800 individuals respond so hopefully we can meet this number
+again this year. 
+
+Once the survey has been completed, we will publish the results for the entire community to see.
+
+The survey should take less than 10 minutes to complete. Thank you in advance for your feedback.
+
+[Survey Form](https://forms.gle/yhn8EzxA98yDFsjh7)
```

Comment:
<img width="400" alt="Screenshot 2023-02-23 at 18 41 02" src="https://user-images.githubusercontent.com/20310935/220871373-a6fb366d-95d9-446f-82a4-04abc3d8fb6c.png">
"Raspbian" -> "Raspberry Pi OS"

Reply:
Change made. Thanks for the feedback.

Reply:
Thank you for updating it. But it seems incorrect...
"Raspbian Pi OS" -> "Raspberry Pi OS"
<img width="400" alt="Screenshot 2023-02-24 at 1 22 53" src="https://user-images.githubusercontent.com/20310935/220968889-7265c940-0f27-465e-9ff7-aa0d57dbcbc8.png">

Reply:
Fixed. Thanks @kazuhitoyokoi

Reply:
Another one, "Windews Desktop" -> "Windows Desktop".

Reply:
Done

Reply:
Thank you! @iskerrett @knolleary